mailmine.py

```diff
@@ -67,6 +68,11 @@
         except phonenumbers.NumberParseException:
             pass
 
+
+counters = collections.defaultdict(collections.Counter)
```

Comment:
This code should be placed in the "I haven't seen this contact before" section at line 53.